app/views/shared/_link_table_row.html.erb

```diff
@@ -1,10 +1,9 @@
-<%= content_tag 'tr',
+<%= content_tag "tr",
     class: [
-      link.first ? 'first-row' : '',
-      link.updated? ? 'success' : ''
-    ].join(' '),
-    data: link.row_data do
-%>
+      link.first ? "first-row" : "",
+      link.updated? ? "success" : "",
+    ].join(" "),
+    data: link.row_data do %>
   <td class="text-center lgsl">
```

Comment:
I'm pretty sure the class names `first-row` and `success` are old formatting styles and we can safely delete them.